staging/src/k8s.io/apimachinery/pkg/api/validate/strfmt_test.go

```diff
@@ -17,9 +17,102 @@ limitations under the License.
 package validate
 
 import (
+	"context"
+	"strings"
 	"testing"
+
+	"k8s.io/apimachinery/pkg/api/operation"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	fldTesting "k8s.io/apimachinery/pkg/util/validation/field/testing"
 )
 
 func TestDNS1123Label(t *testing.T) {
-	// DNSLabel is a trivial wrapper of content.DNS1123Label.
+	ctx := context.Background()
+	fldPath := field.NewPath("test")
+
+	testCases := []struct {
+		name     string
+		label    string
+		wantErrs field.ErrorList
+	}{
+		{
+			name:     "valid label",
+			label:    "valid-label",
+			wantErrs: nil,
+		},
+		{
+			name:     "valid single character label",
+			label:    "a",
+			wantErrs: nil,
+		},
+		{
+			name:     "valid label with numbers",
+			label:    "123-abc",
+			wantErrs: nil,
+		},
+		{
+			name:  "invalid: uppercase characters",
+			label: "Invalid-Label",
+			wantErrs: field.ErrorList{
+				field.Invalid(fldPath, "Invalid-Label", "").WithOrigin("format=dns-label"),
```

Comment:
As with the other test, I think you could just pass nil for the value, but it's not super important.